haas/rest.py

```diff
@@ -123,6 +125,10 @@ def rest_call(method, path, schema):
             argument to appear in both the request body and the path; such
             requests will be rejected prior to invoking the schema.
 
+            Because GET query parameters have no type, ensure that any GET
+            arguments of non-string type implement Schema's "Use" to perform
+            type validation and conversion.
```

Comment:
Get parameters do not "have no type;" their type is string. 

I think this particular warning doesn't need to be here, but you *do* need to update the rest of the description to describe the modified behavior; the preceeding paragraph says path -> body still.